tests/test_extension.py

```diff
@@ -1,22 +1,16 @@
 import os
 import sys
 import unittest
-from sphinx_testing import with_app
+import pytest
 
 
-sys.path.insert(0,
-                os.path.join(os.path.dirname(os.path.abspath(__file__)), '..'))
+PARENT = os.path.join(os.path.dirname(os.path.abspath(__file__)), '..')
+sys.path.insert(0, PARENT)
 
+@pytest.mark.sphinx(buildername='html', srcdir=os.path.join(PARENT, 'docs'))
```

Comment:
I'm curious if this coming built-in with pytest or comes from the Sphinx dependency?

Reply:
From Sphinx.